music21/analysis/discrete.py

```diff
@@ -1024,15 +1024,14 @@ def getPitchSpan(self, subStream) -> Optional[Tuple[pitch.Pitch, pitch.Pitch]]:
 
         >>> s = stream.Stream(note.Rest())
         >>> p.getPitchSpan(s)
-        None
```

Comment:
generally I try to write either

```
>>> print(p.getPitchSpan(s))
None
```

or

```
>>> p.getPitchSpan(s) is None
True
```

It is the difference between doing a statement for side effects alone

```
>>> p.hello = 'hi'
```

and doing something and testing that it returns None.  Of course this does that also, but it's less clear for readers that that's what we're doing.